planner-api/src/main/java/pl/edu/agh/kuce/planner/balance/BalanceController.java

```diff
@@ -0,0 +1,37 @@
+package pl.edu.agh.kuce.planner.balance;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.edu.agh.kuce.planner.auth.Current;
+import pl.edu.agh.kuce.planner.auth.persistence.User;
+import pl.edu.agh.kuce.planner.balance.dto.BalanceDto;
+import pl.edu.agh.kuce.planner.balance.service.BalanceService;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("api/balance")
+public class BalanceController {
+    private final BalanceService balanceService;
+
+    public BalanceController(final BalanceService balanceService) {
+        this.balanceService = balanceService;
+    }
+
+    @PostMapping("/create")
+    public void create(@Current final User user, @Valid @RequestBody final Integer balance) {
```

Comment:
I wonder if this endpoint is neccessary. Shouldn't balance be created when a user account is created? Now, there is a possibility that there is a user who has no balance. Perhaps we can just use 'balance' field in User entity and we don't need Balance entity?

Reply:
I was talking about this with Ela and this change is in progress, you have right, better is to automatically create balance for the user while creating user